com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkPrefab.cs

```diff
@@ -4,21 +4,44 @@
 
 namespace MLAPI.Configuration
 {
+    public enum NetworkPrefabOverride
+    {
+        None,
+        Prefab,
+        Hash
+    }
+
     /// <summary>
     /// Class that represents a NetworkPrefab
     /// </summary>
     [Serializable]
     public class NetworkPrefab
     {
```

Comment:
to reduce surface area even further, I wonder if we could make both `enum NetworkPrefabOverride` and `class NetworkPrefab` `internal` instead of `public`. they're not going to be used by framework users on any APIs we have — and `[Serializable]` attribute will still serialize `NetworkPrefab` instances even if it was `private`, it should be safe to hide them behind `internal` access modifiers.

also minor: I'd suggest naming `NetworkPrefab` as `NetworkPrefabConfig` to indicate/communicate more clearly that it's just a config struct.